src/components/Carrousel.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import compose from 'lodash/flowRight'
+import SwipeableViews from 'react-swipeable-views'
+import { Media, Bd, Img } from 'cozy-ui/transpiled/react/Media'
+import { withBreakpoints } from 'cozy-ui/transpiled/react'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import cx from 'classnames'
+import range from 'lodash/range'
+
+class Carrousel extends React.PureComponent {
+  constructor(props, context) {
+    super(props, context)
+    this.state = { currentIndex: 0 }
+    this.handleChange = this.handleChange.bind(this)
+    this.handlePrev = this.handlePrev.bind(this)
+    this.handleNext = this.handleNext.bind(this)
+  }
+
+  handleChange(i) {
+    this.setState({ currentIndex: i })
+  }
+
+  handlePrev() {
+    this.setState({ currentIndex: Math.max(0, this.state.currentIndex - 1) })
+  }
+
+  handleNext() {
+    this.setState({
+      currentIndex: Math.min(
+        this.props.children.length - 1,
+        this.state.currentIndex + 1
+      )
+    })
+  }
+
+  render() {
+    const { currentIndex } = this.state
+    const {
+      children,
+      className,
+      breakpoints: { isMobile }
+    } = this.props
+    return (
+      <Media className={className}>
+        <Img
+          className={cx('u-c-pointer u-slateGrey', {
+            'u-o-0': currentIndex === 0
```

Comment:
This doesn't prevent from clicking on the element. It is not visible but the user can still interact with it. If we want the element to keep its dimensions so nothing jumps, we can add `visibility: hidden` to it